code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -762,6 +774,12 @@ func (sc *StorageSmartContract) addChallenge(alloc *StorageAllocation,
 	if err != nil {
 		return "", err
 	}
+
+	err = emitAddOrOverwriteAllocation(alloc, balances)
+	if err != nil {
+		return "", common.NewErrorf("blobber_challenge_decode_error",
```

Comment:
blobber_challenge_decode_error ?

Reply:
yes, this seems unusual, I'll return a relevant error.